pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -433,7 +433,7 @@ https://github.com/lvimuser/lsp-inlayhints.nvim/,HEAD,
 https://github.com/Issafalcon/lsp-overloads.nvim/,main,
 https://github.com/ahmedkhalf/lsp-rooter.nvim/,,
 https://github.com/nvim-lua/lsp-status.nvim/,,
-https://github.com/VonHeikemen/lsp-zero.nvim/,main,
+https://github.com/VonHeikemen/lsp-zero.nvim/,HEAD,
```

Comment:
```suggestion
https://github.com/VonHeikemen/lsp-zero.nvim/,v3.x,
```
`v3.x` is the default branch of the repository. Though I think HEAD will simply point to the latest commit of the default branch anyhow.